src/classes/TestRatingController.cls

```diff
@@ -0,0 +1,52 @@
+/**
+ * Created by drang on 10/15/2018.
+ */
+
+@isTest
+public with sharing class TestRatingController {
+
+    @isTest
+    static void testSaveNewRating(){
+
+        Mountains__c testMountain = new Mountains__c(Name = 'Rila');
+        insert testMountain;
+
+        Trial__c testTrial = new Trial__c(Mountains__c = testMountain.Id , Name = 'testTrial');
+        insert testTrial;
+
+        Rate__c testRating = new Rate__c(Trial__c = testTrial.Id , Rating__c = 5, Description__c = 'awesome');
+
+        Test.startTest();
+        RatingController.saveRating(testRating);
+        Test.stopTest();
+
+        Rate__c savedRating = [Select id,Trial__c,Rating__c, Description__c  FROM Rate__c Where Trial__c = :testTrial.Id AND Rating__c = 5 AND Description__c = 'awesome'];
```

Comment:
This query is too much. Since it's test and has it's own set of data, the only rating should be the one inserted during test execution. Simple query is better - check also if values didn't change during insertion (i.e. when you write a trigger afterwards)